plexapi/sync.py

```diff
@@ -1,31 +1,58 @@
 # -*- coding: utf-8 -*-
+"""
+You can work with Mobile Sync on other devices straight away, but if you'd like to use your app as sync-target you
+need to init some variables.
+
+Example:
+    def init_sync():
+        import plexapi
+        plexapi.X_PLEX_PROVIDES = 'sync-target'
+        plexapi.BASE_HEADERS['X-Plex-Sync-Version'] = '2'
+        plexapi.BASE_HEADERS['X-Plex-Provides'] = plexapi.X_PLEX_PROVIDES
+
+        # mimic iPhone SE
+        plexapi.X_PLEX_PLATFORM = 'iOS'
+        plexapi.X_PLEX_PLATFORM_VERSION = '11.4.1'
+        plexapi.X_PLEX_DEVICE = 'iPhone'
+
+        plexapi.BASE_HEADERS['X-Plex-Platform'] = plexapi.X_PLEX_PLATFORM
+        plexapi.BASE_HEADERS['X-Plex-Platform-Version'] = plexapi.X_PLEX_PLATFORM_VERSION
+        plexapi.BASE_HEADERS['X-Plex-Device'] = plexapi.X_PLEX_DEVICE
+        plexapi.BASE_HEADERS['X-Plex-Model'] = '8,4'
+        plexapi.BASE_HEADERS['X-Plex-Vendor'] = 'Apple'
+
+    You have to fake platform/device/model because transcoding profiles are hardcoded in Plex, and you obviously have
+    to explicitly specify that your app supports `sync-target`
+"""
+
 import requests
-from plexapi import utils
+import plexapi
 from plexapi.exceptions import NotFound
+from plexapi.base import PlexObject
 
 
-class SyncItem(object):
-    """ Sync Item. This doesn't current work. """
-    def __init__(self, device, data, servers=None):
-        self._device = device
-        self._servers = servers
-        self._loadData(data)
+class SyncItem(PlexObject):
+    TAG = 'SyncItem'
+
+    def __init__(self, server, data, initpath=None, clientIdentifier=None):
+        super(SyncItem, self).__init__(server, data, initpath)
+        self.clientIdentifier = clientIdentifier
 
     def _loadData(self, data):
         self._data = data
```

Comment:
There should be a documentation header on head class in this file explaining all the attributes and why they exist. A good example format is the video.py file.

https://github.com/pkkid/python-plexapi/blob/master/plexapi/video.py